web/yo/app/scripts/controllers/gene.js

```diff
@@ -2306,15 +2306,18 @@ angular.module('oncokbApp')
              * @param obj: corresponding object
              * @param uuids: the array that you want to append uuids to. usually, pass in a empty array
              * @param inside: boolean value, set it to true to exclude its own uuid from getting collected. Otherwise this function will collect all of the uuids
+             * @param evidenceUUIDsOnly: boolean value, set it to true to indicate only want to collect evidences uuid inside one section. If not specified, will return all UUIDs besides evidences UUIDs
              * */
-            function collectUUIDs(type, obj, uuids, inside) {
+            function collectUUIDs(type, obj, uuids, inside, evidenceUUIDsOnly) {
                 if (type === 'mutation') {
                     if (!inside) {
                         uuids.push(obj.name_uuid.getText());
                     }
                     uuids.push(obj.oncogenic_uuid.getText());
                     uuids.push(obj.effect_uuid.getText());
-                    uuids.push(obj.description_uuid.getText());
+                    if (!evidenceUUIDsOnly) {
```

Comment:
Just curious if the evidenceUUIDsOnly is null or undefined, then it should behave as false?

Reply:
@zhx828 Yes, that is correct. This function is used to collect all UUIDs inside a section by default